README.md

```diff
@@ -5,28 +5,48 @@ Mongodb
 
 ## Immediate Roadmap
 
-  * Bang and non-bang `Mongo` functions
-  * Documentation
-  * Move low-level API to `Connection` and have `Mongo` be the next-gen driver API
-    - Split database and collection actions?
-  * Move BSON encoding to client process
-    - Make sure requests don't go over the 16mb limit
-  * Replica sets
-    - Block in client (and timeout) when waiting for new primary selection
-  * New 2.6 write queries and bulk writes
-  * Cursors
-  * Reconnect backoffs with https://github.com/ferd/backoff
-  * Lazy connect
+* Bang and non-bang `Mongo` functions
+* Documentation
+* Move low-level API to `Connection` and have `Mongo` be the next-gen driver API
+  - Split database and collection actions?
+* Move BSON encoding to client process
+  - Make sure requests don't go over the 16mb limit
+* Replica sets
+  - Block in client (and timeout) when waiting for new primary selection
+* New 2.6 write queries and bulk writes
+* Cursors
+* Reconnect backoffs with https://github.com/ferd/backoff
+* Lazy connect
```

Comment:
Please don't unindent lists. The markdown spec says lists should be indented with 2 spaces.